### PR TITLE
docs: resubmit `Manage Issues` documentation

### PR DESCRIPTION
**Pull request by snitin315**
#### Prerequisites checklist

- [X] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
Refers #17225

Resubmit changes from https://github.com/eslint/eslint/pull/17157, https://github.com/eslint/eslint/pull/17084 and https://github.com/eslint/eslint/commit/970ef1c868235a58297682513842f1256cdfbd03

#### Is there anything you'd like reviewers to focus on?

There was a typo in the original PR(https://github.com/eslint/eslint/pull/17157), I've fixed that too (`assessement` -> `assessment `)

### CODE REVIEW

**Review by amareshsm — APPROVED** [review]
LGTM. Waiting for 2nd review.

**Review by mdjermanovic — APPROVED** [review]
The changes LGTM, but I'm not sure if this should rather have been submitted by the original author of https://github.com/eslint/eslint/commit/970ef1c868235a58297682513842f1256cdfbd03, https://github.com/eslint/eslint/commit/1fea2797801a82a2718814c83dad641dab092bcc, and https://github.com/eslint/eslint/commit/7f183e020579380fa57473caaf9ed154470c25b3 (@nzakas all three) so I'd like @nzakas to verify.

**Review by nzakas — APPROVED** [review]
As the original author, I give permission for these changes to be merged.

**Pull request merged**
